src/main/java/com/netflix/blitz4j/LoggingConfiguration.java

```diff
@@ -318,7 +322,19 @@ public synchronized void clearProperty(Object source, String name, Object value,
      * com.netflix.config.PropertyListener#configSourceLoaded(java.lang.Object)
      */
     public void configSourceLoaded(Object source) {
-        Properties props = ConfigurationConverter.getProperties(ConfigurationManager.getConfigInstance());
+        AbstractConfiguration config = ConfigurationManager.getConfigInstance();
+        Properties props = new Properties();
+
+        char delimiter = config.getListDelimiter();
+
+        for (Iterator<String> keys = config.getKeys(LOG4J_PREFIX); keys.hasNext();) {
```

Comment:
This lib builds with java 8, so we're safe to do
```
for (String key: config.getKeys(LOG4J_PREFIX)) {
...
}

Reply:
Unfortunately config.getKeys returns an iterator which doesn't support for-each

Reply:
😅 I always forget one needs an iterABLE